libafl/src/bolts/llmp.rs

```diff
@@ -851,6 +853,13 @@ where
             }
             unmap_until_excl += 1;
         }
+
+        if unmap_until_excl == 0 && self.out_maps.len() > 3 {
+            // We send one last information to the broker before quitting.
+            self.send_buf(LLMP_SLOW_RECEIVER_PANIC, &[]).unwrap();
+            panic!("The receiver/broker could not process our sent llmp messages in time. Either we're sending too many messages too fast, the broker got stuck, or it crashed. Giving up.");
```

Comment:
Does this already implement a way to disable this behaviour for debuggin?

Reply:
It does not, read my reasoning here:
https://github.com/AFLplusplus/LibAFL/pull/230#issuecomment-882636414

Reply:
Btw, now that we have a `LLMP_CFG_MAX_PENDING_UNREAD_PAGES` constant, you can just set the value to a very high number (not that it would make any sense in real life, though :) )